### ISSUE TITLE
Wordpress: Recreate the FOSSASIA start page

### ISSUE THREAD

**Issue by mariobehling** [fossasia/Codeheat#11]
This issue can be taken by multiple CodeHeat participants.

### OBJECTIVE
The FOSSASIA start page at fossasia.org is currently hosted on Github. The goal is to host the start page on Wordpress in future. Therefore please recreate the page on a self-hosted Wordpress using the OceanWP theme. After you finished please post the link here and provide a way to import it to the FOSSASIA Wordpress.

### REQUIREMENTS
- Self-hosted Wordpress
- Use OceanWP theme
- Ensure it works nicely on mobile screens
- Provide an export option
- Register as a CodeHeat participant if you haven't already done so https://codeheat.org/register

### EXPECTED OUTCOME
The expected out is a Wordpress hosted start page similar to the current start page of FOSSASIA.org. It should work nicely on desktop and mobile screens.

### HOW TO SUBMIT YOUR WORK
Provide a export option of the page.

### LINKS
- If you have any questions, please reach out to our Gitter channel: https://gitter.im/fossasia/fossasia

**Comment by Cvr421** [comment]
hello @mariobehling can i work on this issue

**Comment by abhishekd358** [comment]
Please assign this issue to me

**Comment by mariobehling** [comment]
@Cvr421 @abhishekd358 Thank you for your interest! You can solve this issue separately or you can collaborate and solve it together. Please set up a Wordpress with the [OceanWP](https://oceanwp.org/) theme somewhere and create a start page using the text and images from https://github.com/fossasia/fossasia.org.

**Comment by rupal16** [comment]
I can work on this issue

**Comment by mariobehling** [comment]
@rupal16 Please go ahead as well.

**Comment by ritwikdurga** [comment]
Hello @rupal16 is this still available so that i can work on this too ?

**Comment by n-keerthi-gayathri** [comment]
Hello @mariobehling, can I work on this issue?

**Comment by mariobehling** [comment]
Yes, this issue is still available. Please proceed with it. Thanks

**Comment by Satarupa22-SD** [comment]
please assign this issue to me as well, thanks!

**Comment by hpdang** [comment]
Please note this issue can be taken by multiple participants. It will remain open and accessible to all participants until a successful PR is submitted.

**Comment by weshaan** [comment]
imma start this

**Comment by weshaan** [comment]
guys, for this I need the creator plan on Wordpress, will you(fossasia) be funding the premium subscription ?

**Comment by n-keerthi-gayathri** [comment]
@weshaan are you willing to collaborate?

**Comment by weshaan** [comment]
> @weshaan are you willing to collaborate?

I am looking forward to participate and if possible do good in codeheat

**Comment by rushikedhar04** [comment]
Hey @mariobehling is this issue still available?

**Comment by weshaan** [comment]
> Hey @mariobehling is this issue still available?

should be

**Comment by Rithi-LAB** [comment]
Can I work on this issue?
